server/routes/dataInsights/punishmentsTab/punishments.test.ts

```diff
@@ -34,6 +34,12 @@ beforeEach(() => {
       chartEntries: [],
     })
   )
+  chartApiService.getLastModifiedChart.mockResolvedValue(
+    testData.chartLastUpdatedResult({
+      chartName: '4c',
```

Comment:
In `punishmentsTabPage.ts` we pass `4a` so good to make this `chartName: '4a'` ?